opentelemetry/src/main/scala/zio/telemetry/opentelemetry/tracing/Tracing.scala

```diff
@@ -747,15 +766,22 @@ object Tracing {
                            .setAllAttributes(attributes)
                            .setSpanKind(spanKind)
                            .setStartTimestamp(nanos, TimeUnit.NANOSECONDS)
+                           .addLinks(links)
                            .startSpan()
                        )
             } yield (endSpan(span), parentCtx.`with`(span))
 
+          implicit class SpanBuilderOps(spanBuilder: SpanBuilder) {
+            def addLinks(links: Seq[SpanContext]): SpanBuilder =
+              links.foldLeft(spanBuilder) { case (builder, link) => builder.addLink(link) }
+          }
+
```

Comment:
It makes sense to hide it from the end-user by adding a `private` modifier.

Reply:
done